### DISCUSSION TITLE
自定义程度很高的主题。

### DISCUSSION THREAD

**Discussion by LeiWang1999** [YunYouJun/hexo-theme-starry#129] [Show 分享展示]
blog: LeiWang1999‘s Blog
url: https://leiblog.wang
avatar: https://leiblog.wang/images/avatar_1.jpg
desc: The warehouse of LeiWang1999

感谢 云游君 up提供的主题，很棒！

**Comment by YunYouJun** [comment]
Thanks. Have fun!

**Reply by YunYouJun** [reply]
站点定制的很漂亮！稍后会加入示例站点中！

**Reply by LeiWang1999** [reply]
:DD
